### PR TITLE
PlaySound changed a bit in 7.3.0, no longer accepts strings

### PR DESCRIPTION
**Pull request by mdklein**
PlaySound now only will accept a Soundkit ID. I replaced the strings with the Soundkit ID definitions.

### CODE REVIEW

**Comment by onyxraven** [comment]
looks like this should be the same as https://github.com/AdiAddons/AdiBags/issues/217 ?

**Comment by tcprescott** [comment]
These changes fix the addon for me.  Thanks for the pull request!

**Comment by onyxraven** [comment]
@tcprescott @casz @BillRiess I'm a newcomer to addons (I did a _tiny_ bit back in 1.0 days). Is there a guide on the best way to load up this branch myself?

**Comment by BillRiess** [comment]
@onyxraven https://github.com/mdklein/AdiBags/archive/master.zip

File: AdiBags.toc [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # You should have received a copy of the GNU General Public License
 # along with AdiBags.  If not, see <http://www.gnu.org/licenses/>.
 
-## Interface: 70200
+## Interface: 70300
```
Comment (onyxraven) [inline AdiBags.toc]: does the Config toc need bumped too?

Comment (mdklein) [reply AdiBags.toc]: Yes, I forgot about it. Thanks.

**Comment by onyxraven** [comment]
Hm, I downloaded master and copied the files over - I see the addon in the list but opening bags no longer brings up the adiBags interface (albiet with errors), it's opening the classic bags instead :/

**Comment by Zarggg** [comment]
@onyxraven: Make sure you delete the `Adibags` and `Adibags_Config` directories from your `ADDONS` directory before copying over the replacement.

**Comment by onyxraven** [comment]
@Zarggg thanks, I got it working - deleted everything *except* the libs folder.

**Pull request merged**